### PR TITLE
Th2 4490 add data map stream method

### PR DESCRIPTION
**Pull request by xHacka**

### CODE REVIEW

**Review by ConnectDIY — CHANGES_REQUESTED** [review]
I've added TODO comments

**Review by ConnectDIY — CHANGES_REQUESTED** [review]
Everything is ok, add please, release notes, as our workflow required

File: release_notes.md [hunk 1]
```diff
@@ -118,7 +118,7 @@
 
 ## User impact and migration instructions
 
-1. [I] The "metadataOnly" query parameter will now always be set to False in the get_X_from_data_provider methods. It’s
+1. [I] The "metadataOnly" query parameter will now always be set to False in the get_X_from_data_provider methods. Its
```
Comment (ConnectDIY) [inline release_notes.md:121]: ??

File: release_notes.md [hunk 2]
```diff
@@ -267,141 +271,161 @@ This release is not required any additional steps to use.
...
 
-      **Example**
+**Example**
 
-      ```
-      pip install th2-data-services[rdp5]
-      ```
+   ```
+   pip install th2-data-services[rdp5]
+   ```
```
Comment (ConnectDIY) [inline release_notes.md:383]: Were all these changes made by some auto script, or you just solved to change it?

File: release_notes.md [hunk 3]
```diff
@@ -267,141 +271,161 @@ This release is not required any additional steps to use.
...
-   | lwdp1    | latest version of lwdp (version 1) |
-   | lwdp2    | latest version of lwdp (version 2) |
+| provider |              version               |
+|----------|:----------------------------------:|
+| lwdp     |       latest version of lwdp       |
+| lwdp1    | latest version of lwdp (version 1) |
+| lwdp2    | latest version of lwdp (version 2) |
 
 3. [TH2-4493] Adapter interface got handle_stream method.
+4. [TH2-4490] Added `map_stream` method to Data. 
```
Comment (ConnectDIY) [inline release_notes.md:431]: I hope, you agree that it's not all. map_stream also can take Adapters, not only map function.

We also have to add some examples to our demo example script

**Review by ConnectDIY — CHANGES_REQUESTED** [review]

File: release_notes.md (hunk 1 above)
Comment (xHacka) [reply release_notes.md:121]: Must be `pre-commit`, I havent touched it 😅

File: release_notes.md (hunk 2 above)
Comment (xHacka) [reply release_notes.md:383]: Except table format (which was a little broken) pre-commit formatted it

**Review by ConnectDIY — APPROVED** [review]

**Pull request merged**